README.rst

```diff
@@ -273,7 +274,8 @@ Or, if ``virtualenv`` is not installed on your system:
     $ python virtualenv.py develop    # Create virtual environment
     $ source develop/bin/activate     # Change default python to virtual one
     (develop)$ pip install -r requirements.txt  # Install requirements for 'Vocablary' in virtual environment
-    (develop)$ git clone https://github.com/prodicus/vocabulary.git
+    (develop)$ cd vocabulary
+    (develop)$ pip install -r requirements.txt  # Install requirements for 'Vocabulary' in virtual environment
```

Comment:
yes. my bad. thanks for pointing it out. will correct  it and send a pull request